onnxscript/optimizer/_constant_folding.py

```diff
@@ -944,21 +944,21 @@ def new_constant(self, node: ir.Node, value) -> ir.Node | None:
         tensor.name = irvalue.name
         irvalue.const_value = tensor
 
-        if value.nbytes > self.output_size_limit:
+        if value.size > self.output_size_limit:
```

Comment:
IIRC, there is probably documentation about the output/input size limit parameters that should also be updated to reflect this.

Reply:
I copied but forgot to paste: https://github.com/titaiwangms/onnx-script/blob/42fafa475d1d4e44b53b3cecc1ab914b07d2c00e/onnxscript/optimizer/_constant_folding.py#L1193-L1198

Reply:
Done

Reply:
I don't see the changes in documentation here

Reply:
I think below